components/compliance-service/compliance.go

```diff
@@ -554,22 +554,36 @@ func setup(ctx context.Context, connFactory *secureconn.Factory, conf config.Com
 	} else {
 		// get the authn-service connection
 		logrus.Debugf("compliance setup, dialing authn-service(%s)", conf.InspecAgent.AuthnTarget)
-		authConn, err = connFactory.DialContext(timeoutCtx, "authn-service", conf.InspecAgent.AuthnTarget,
+		authnConn, err = connFactory.DialContext(timeoutCtx, "authn-service", conf.InspecAgent.AuthnTarget,
 			grpc.WithBlock())
-		if err != nil || authConn == nil {
+		if err != nil || authnConn == nil {
 			err = errors.New("compliance setup, error grpc dialing to authn aborting...")
 			return err
 		}
 		// get the authn client
-		authClient := auth.NewTokensMgmtClient(authConn)
-		if authClient == nil {
+		authnClient := authn.NewTokensMgmtClient(authnConn)
+		if authnClient == nil {
 			logrus.Errorf("serveGrpc got nil for NewTokensMgmtClient: %s", err)
 			return err
 		}
+		// get authz connection
+		authzConn, err = connFactory.DialContext(timeoutCtx, "authz-service", fmt.Sprintf("%s:%d", conf.Authz.HostBind, conf.Authz.Port),
+			grpc.WithBlock())
```

Comment:
we already have the authz host/port info bc we use it during project tagging and updating